v22.2/security-reference/scram-authentication.md

```diff
@@ -93,17 +101,21 @@ SELECT username, "hashedPassword" FROM system.users WHERE username='hypothetical
+The procedure documented here, creating a SCRAM-SHA-256-authenticated user by passing a *cleartext* password to the CockroachDB client, achieves the benefit of not requiring the cleartext password to be transmitted over the network; this offers good protection from replay attacks in case attackers bypass TLS. But it does not achieve full separation of concerns by isolating the cleartext password to a delegated service. The necessary additional steps to achieve this are documented in [Implement strict isolation of cleartext credentials](#implement-strict-isolation-of-cleartext-credentials).
 {{site.data.alerts.end}}
 
-### Migrating existing users/roles to SCRAM-SHA-256 authentication
+### Migrate existing users/roles from cleartext passwords to SCRAM-SHA-256 authentication
 
 #### Eventual migration
 
-It is possible to automatically convert the records for previously created users from plaintext passwords to SCRAM-compatible hash encodings. This will cause CockroachDB to use SCRAM authentication when client apps connect to CockroachDB, provided that the cluster's authentication configuration has also been set to accept SCRAM-SHA-256.
+It is possible to automatically convert the records for previously created users from cleartext passwords to SCRAM-compatible hash encodings. This will cause CockroachDB to use SCRAM authentication when client apps connect to CockroachDB, provided that the cluster's authentication configuration has also been set to accept SCRAM-SHA-256.
 
 To convert existing users to SCRAM-SHA-256, enable the `server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled` cluster setting. When this setting is enabled, the conversion occurs the first time a client app connects with the previously defined password. During that first connection, the previous mechanism will be used to establish the connection, and then CockroachDB will re-encode the password using the SCRAM algorithm. Subsequent connections will then use the SCRAM handshake for authentication.
 
+{{site.data.alerts.callout_info}}
+In CockroachDB v22.x and above, the `server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled` cluster setting is enabled by default. Because `user_login.password_encryption` also defaults to `scram-sha-256`, this means that by default, any user who still uses cleartext passwords will be migrated to SCRAM-SHA-256 authentication. To prevent this automatic migration, set `server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled` to `false` before upgrading to CockroachDB v22.x.
```

Comment:
```suggestion
In CockroachDB v22.x and above, the `server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled` [cluster setting](cluster-settings.html) is enabled by default. Because `user_login.password_encryption` also defaults to `scram-sha-256`, this means that by default, any user who still uses cleartext passwords will be migrated to SCRAM-SHA-256 authentication. To prevent this automatic migration, set `server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled` to `false` before upgrading to CockroachDB v22.x.
```